core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -760,6 +759,8 @@ import scala.util.control.NonFatal
 
       override def onPartitionsRevoked(partitions: java.util.Collection[TopicPartition]): Unit = ()
 
+      override def onPartitionsLost(partitions: java.util.Collection[TopicPartition]): Unit = ()
```

Comment:
ah, Scala 3 doesn't understand `default` method from Java interface
From ConsumerRebalanceListener:
```
default void onPartitionsLost(Collection<TopicPartition> partitions) {
        onPartitionsRevoked(partitions);
    }
```

@SethTisue do you think this should be reported upstreams or is it intentional?

Reply:
having a look...

Reply:
Incidentally one of the first things I tried here was Scala 3.2.2 instead of 3.1.3. It turned out not to make any difference in this case, but regardless, I'd recommend moving to 3.2.2 if you can, as bugfixes are always landing. Also 3.3.0 will be out soon (they're at 3.3.0-RC3).

Reply:
Thanks for looking into it @SethTisue 
I think we should stick to the lower 3.1 version since we want to support as many versions as possible (if the cost isn't too high). Scala 3.x is backwards compatible but not forwards compatible so if we bump to lates we would exclude users of earlier versions.

Reply:
okay, but lots of projects are taking the opposite view, that it's fine and normal for users to need to upgrade their Scala 3 minor version when upgrading a dependency. that's also the view that the Scala Center is encouraging library maintainers to take. and we've already merged the 3.2 upgrade even in foundational libraries such as scala-xml and scala-collection-compat.

Scala 3.3 will be the first "long term support" version of Scala 3. you might consider adopting your more conservative view once 3.3 is available, but take more the liberal view until then. 3.1 isn't getting bugfixes anymore, and 3.2 isn't either, now that 3.3 is imminent.

Reply:
ok, good to know. We should anyway try to use the same version in all Akka projects as much as possible, so such change would not start here.

Reply:
Do you want to take the opportunity and bump the scala version in all projects before the next release?